src/apiClient.tsx

```diff
@@ -59,31 +110,117 @@ export interface OrganizationCreateRequest {
 }
 
 export interface OrganizationResponse {
-  id: number;
   full_name?: string;
   short_name: string;
   description?: string;
   url?: string;
+  id: number;
   created_at: string;
 }
 
+export interface OrganizationUpdateRequest {
+  full_name?: string;
+  short_name: string;
+  description?: string;
+  url?: string;
+}
+
 export interface RequirementResponse {
-  id: number;
   name: string;
   description: string;
+  id: number;
   created_at: string;
 }
 
+export interface RequirementsCreateRequest {
+  name: string;
+  description: string;
+}
+
+export interface RequirementsUpdateRequest {
+  name: string;
+  description: string;
+  id: number;
+}
+
+export interface VaccineAvailabilityCreateRequest {
+  numberAvailable?: number;
+  numberTotal?: number;
+  date?: string;
+  vaccine?: number;
+  inputType?: InputTypeEnum;
+  tags?: string;
+  location: number;
+}
+
 export interface VaccineAvailabilityExpandedResponse {
-  id: string;
-  numberAvailable: number;
+  numberAvailable?: number;
   numberTotal?: number;
-  date: string;
+  date?: string;
   vaccine?: number;
-  inputType: InputTypeEnum;
+  inputType?: InputTypeEnum;
   tags?: string;
-  created_at: string;
+  id: string | number;
```

Comment:
IMPORTANT NOTE TO WHOMEVER GENERATES THE API CLIENT: this id needs to be MANUALLY altered after client generation. `restful-react` types this to `{}` which is not correct. The source swagger JSON has this:

```
id:
          title: Id
          anyOf:
            - type: string
              format: uuid
            - type: integer
```
So basically this is an int or a guid, all good... But the `{}` that gets generated basically means any non-nullish value, which is wrong. Will file an issue with `restful-react`

Reply:
This is pretty unfortunate. It looks somewhat related to this issue https://github.com/contiamo/restful-react/issues/317

Is there any way we can get the backend to return a single type for this ID? It seems like it should just be a string type like the other IDs being returned in the schema.

This manual step of updating the ID in the client here will be very easy to miss.

Reply:
It won't be easy to miss - compilation will fail. It just won't be obvious as to how to resolve it, lol. Hrm, maybe a README update? Not perfect, but might help.

Reply:
Oh yeah, a failing build makes it more obvious. A readme update would probably help to resolve in the interim.

Still wondering why it's typed like this on the backend. Any insight @egamble01?